initializr-generator/src/test/java/io/spring/initializr/generator/buildsystem/maven/MavenBuildTests.java

```diff
@@ -110,4 +114,29 @@ void mavenPluginExtensionsCanBeLoaded() {
 		assertThat(testPlugin.isExtensions()).isTrue();
 	}
 
+	@Test
+	void mavenResourcesCanBeLoaded() {
```

Comment:
One test is not enough I am afraid, look at the other tests in this class for inspiration.

Reply:
I've added some tests, see if its enough